### PR TITLE
fixed the 404 handler redirecting issue

### PR DESCRIPTION
**Pull request by chiangkeith**
fixed the issue what if slot dom doesn't be rendered

gave an err handler for dfp

open eslint and run lint before every single commit

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -201,29 +200,29 @@ export default {
...
         this.viewport = document.querySelector('body').offsetWidth
       }
-    },
+    }
   },
   metaInfo () {
-    let title = '鏡週刊 Mirror Media'
-    let description = '鏡傳媒以台灣為基地，是一跨平台綜合媒體，包含《鏡週刊》以及下設五大分眾內容的《鏡傳媒》網站，刊載時事、財經、人物、國際、文化、娛樂、美食旅遊、精品鐘錶等深入報導及影音內容。我們以「鏡」為名，務求反映事實、時代與人性。'
+    const title = '鏡週刊 Mirror Media'
+    const description = '鏡傳媒以台灣為基地，是一跨平台綜合媒體，包含《鏡週刊》以及下設五大分眾內容的《鏡傳媒》網站，刊載時事、財經、人物、國際、文化、娛樂、美食旅遊、精品鐘錶等深入報導及影音內容。我們以「鏡」為名，務求反映事實、時代與人性。'
```
Comment (hcchien) [inline src/views/Home.vue]: can we put these const to the const file?

File: src/views/List.vue [hunk 2]
```diff
@@ -802,8 +775,11 @@ export default {
...
     }
-    if(!ogTitle && process.env.VUE_ENV === 'server') {
-      throw { massage : 'Page Not Found', code: '404' }
+    if (!ogTitle && process.env.VUE_ENV === 'server') {
+      const e = new Error()
+      e.massage = 'Page Not Found'
+      e.code = '404'
+      throw e
     }
     const title = ogTitle + ' - 鏡週刊 Mirror Media'
```
Comment (hcchien) [inline src/views/List.vue]: The title should be put in the const file.

File: src/views/Home.vue (hunk 1 above)
Comment (chiangkeith) [inline src/views/Home.vue]: yes, no prob

File: src/views/List.vue (hunk 2 above)
Comment (chiangkeith) [inline src/views/List.vue]: would implement it

**Pull request merged**